tasks/cluster_agent.py

```diff
@@ -40,6 +44,24 @@ def build(ctx, rebuild=False, build_include=None, build_exclude=None, race=False
         skip_assets,
     )
 
+    policies_version = "master"
```

Comment:
should we make this configurable? i.e could it be useful to package a custom policies version to build a custom agent version for example?

Reply:
Yes, definitely. Made it configurable in the task.